src/components/DatePicker/index.js

```diff
@@ -0,0 +1,73 @@
+import React, { Component, Fragment } from 'react';
+import { DatePicker } from 'antd';
+import { connect } from 'dva';
+import moment from 'moment';
+import styles from './index.less';
+
+const { RangePicker } = DatePicker;
+
+@connect(({ global, datastore }) => ({
+  indices: datastore.indices,
+  selectedDateRange: global.selectedDateRange,
+}))
+class AntdDatePicker extends Component {
+  handleChange = (dates, dateStrings) => {
```

Comment:
`dates` isn't being referenced in `handleChange`, do we still need this param here?

Reply:
@gurbirkalsi Actually, when the user selects a date from the datepicker, there are two values returned by the event: 
- `dates`: which are moment objects
- `dateStrings`: which are moment formatted strings.

If I just reference one parameter here, then it automatically behaves as the `dates` in the default parameter

Reply:
Thanks for the clarification!